pkg/kubectl/custom_column_printer_test.go

```diff
@@ -115,8 +122,19 @@ func TestNewColumnPrinterFromSpec(t *testing.T) {
 			t.Errorf("[%s] unexpected error: %v", test.name, err)
 			continue
 		}
+		if test.noHeaders {
+			buffer := &bytes.Buffer{}
 
-		if !reflect.DeepEqual(test.expectedColumns, printer.Columns) {
+			printer.PrintObj(&api.Pod{}, buffer)
```

Comment:
How about reusing the `DeepEqual` check, and set `test.expectedColumns` (which should have `Header: ""`)?